Magento_Theme/templates/html/slider.phtml

```diff
@@ -173,7 +173,7 @@ $sliderContentAfter = $block->getContentAfter();
 
         <?php if ($sliderContentAfter): ?>
             <div class="slider__after">
-                <?= $sliderContentAfter ?>
+                <?= $block->escapeHtml($sliderContentAfter) ?>
```

Comment:
I think here we should use escapeHtml or noEscape.
@pawel-snowdog what kind of content (plain text or HTML) do we have here?

Reply:
It shouldn't be escaped